runtime/runtime/src/lib.rs

```diff
@@ -1765,7 +1766,7 @@ impl Runtime {
             // want to store invalid receipts in state as delayed.
             validate_receipt(
                 &processing_state.apply_state.config.wasm_config.limit_config,
-                receipt,
+                &receipt,
```

Comment:
Is the `&` needed? I think `receipt` is already a `&Receipt`. Same with the one below.